pytest_invenio/user.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of pytest-invenio.
+# Copyright (C) 2022 CERN.
+#
+# pytest-invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+
+"""Helper class for creating and using user fixtures."""
+
+from copy import deepcopy
+
+from flask_principal import Identity, identity_changed
+
+
+class UserFixtureBase:
+    """A user fixture for easy test user creation."""
+
+    def __init__(self, email=None, password=None, active=True):
+        """Constructor."""
+        self._email = email
+        self._active = active
+        self._password = password
+        self._identity = None
+        self._user = None
+        self._client = None
+
+    #
+    # Creation
+    #
+    def create(self, app, db):
+        """Create the user."""
+        from flask_security.utils import hash_password
```

Comment:
Imports inside to allow pytest-invenio to work without having Invenio-accounts installed.